jxl/META.yml

```diff
@@ -0,0 +1,3 @@
+spec: https://jpeg.org/jpegxl/workplan.html
```

Comment:
For W3C and WHATWG specs we have a way to pick the directory name in WPT based on the spec URL. Here there's no rule to follow, but I would still suggest jpegxl/ rather than jxl/. That would match part of this URL, and also https://caniuse.com/jpegxl.

Reply:
done